app/etl/items/importers/quality_metrics.rb

```diff
@@ -14,7 +14,7 @@ def initialize(item_id, _ = {})
 
   def run
     item = Dimensions::Item.find(@item_id)
-
+    return unless item.get_content
     quality_metrics = content_quality_service.run(item.get_content)
```

Comment:
In general it is a bit better to have only one exit point.
What about expressing this logic with:

```ruby
if item.get_content
  # do something
end
```